polkit: update to 126

KexyBiscuit opened the pull request:
Topic Description
-----------------

- polkit: update to 126
    Co-authored-by: Kexy Biscuit a.k.a. 百合ヶ咲るる \(@KexyBiscuit\) <kexybiscuit@outlook.com>

Package(s) Affected
-------------------

- polkit: 1:126

Security Update?
----------------

No

Build Order
-----------

```
#buildit polkit
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

aosc-buildit-bot committed:
polkit: update to 126

Co-authored-by: Kexy Biscuit a.k.a. 百合ヶ咲るる (@KexyBiscuit) <kexybiscuit@outlook.com>

xry111 commented:
Oh, I've removed the `js_engine` option in https://github.com/polkit-org/polkit/pull/494.

xry111 committed:
polkit: Remove -Djs_engine

I removed it in https://github.com/polkit-org/polkit/pull/494 as now
only duktape is supported.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>

MingcongBai reviewed (approved):
Tested good with oma.
